### DISCUSSION TITLE
i need a exploit for this exstention

### DISCUSSION THREAD

**Discussion by shadowxmisty** [3kh0/ext-remover#793] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
there is a exstention called aristole student... and nobody has a removal for it... just in case here is the ext id ¨ ckafplffgnanahfgimapppooplfcfnpm ¨ i would very much love it if you could make  exploit for removing this, its even worse then go guardian

**Comment by Hilololsusy** [comment, marked as answer]
For deextensify, check the main repo here:https://github.com/ading2210/dextensify and make an issue
I use chrome-signin webview. It seems to alwalys work for me. However I dont have the same one

**Reply by AshtonDavies** [reply]
Yeah do that, because Dextensify sucks.

**Reply by DankFireR** [reply]
Indeed it does, it does suck because it just freezes whenever I click it.

**Reply by Hilololsusy** [reply]
I would use LTMEAT but network block, and Im not willing to risk upgrading my school Chromebook to important a second ONC file.

**Reply by Hilololsusy** [reply]
does not work, I have tried

**Comment by shadowxmisty** [comment]
aristole student
